### PR TITLE
Show icon funding widget mobile

### PR DESCRIPTION
**Pull request by sarahcrack**
### Trello card

https://trello.com/c/yc3LHedm/6628-show-icon-on-funding-widget-on-mobile

### Context

Currently the money icon contained within the funding widget on the bursaries and scholarship page disappears when the screen shrinks to mobile size - the icon needs to stay on screen and not disappear, so that the component still stands out to users.

### Changes proposed in this pull request

SimpleComponent (rendered within the funding widget) styling adapted to have desired effect.

### Guidance to review

- Please check the [scholarships and bursaries](https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries) page to make sure the funding widget/icon behaves as expected on different screen sizes.

Screenshot 2024-09-27 at 15 36 23
Screenshot 2024-09-27 at 15 33 37

- The SimpleComponent in question is also utilised on the [homepage](https://getintoteaching.education.gov.uk/) (salaries banner) but the changes don't appear to have affected the banner due to some overriding CSS; an extra pair of eyes to check this would be appreciated (the money icon should still disappear on smaller screen sizes here). 

Screenshot 2024-09-27 at 15 31 13
Screenshot 2024-09-27 at 15 35 21

### CODE REVIEW

**Review by spencerldixon — APPROVED** [review]
LGTM 👍

**Comment by gemmadallmandfe** [comment]
Looks good to me - tested on mobile as well as in emulator, looking at both bursaries page (funding widget) and home page (salaries box)

However, @jenhadfield-dfe should review as the design expert before this is merged!

**Comment by jenhadfield-dfe** [comment]
looks good to me!

**Pull request merged**
